packages/mui/src/range-pickers/wrapper.tsx

```diff
@@ -0,0 +1,165 @@
+import { Button, Grid } from '@mui/material';
+import { useLocaleText } from '@mui/x-date-pickers/internals';
+import { endOfDay, startOfDay } from 'date-fns-jalali';
+import * as React from 'react';
+import { ThemeColorSwapper, ThemeColorSwapperProps } from 'src/shared';
+import { PickerTypes, TimeRange } from 'src/shared/pickers';
+import { DatePicker, DateTimePicker, TimePicker } from 'src/simple-pickers';
+import { WrappedPickerProps } from 'src/simple-pickers/wrapper';
+import { Styles } from '../react-types';
+import { makeStyles } from '../tss-mui';
+
+type PickerProps<P extends PickerTypes, In, Out> = Omit<
+  WrappedPickerProps<P, In, Out>,
+  'multiLocale' | 'value' | 'onChange'
+>;
+
+type Props<P extends PickerTypes, In, Out> = {
+  value: TimeRange;
```

Comment:
`TimeRange<Out>`

Reply:
We will postpone it to another PR! Many files should be touched for this to work.